### ISSUE TITLE
Idiosyncratic language

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7344]
**Preferred term label:**
Idiosyncratic Language

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Abnormal verbal communication disorder

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by bcoleman96** [comment]
Created as HP:4000073

**Comment by pnrobinson** [comment]
@bcoleman96 -- the id is actually HP:4000074
